modules/sync/planetscale/vitess/cas/2f2ac66b50ed2f50290a7124dc68b44c7676813c38d13e5d7e6ef2ac69430896d077ce13089a984634634306d21e22337f9ec2feb08db6e7ec06a0ed68a3e1c3

```diff
@@ -0,0 +1,93 @@
+shake256:29620803f020591ea9429737b2295ad159e8a46674eac5e629d949f292a9d14e1ae6fd7e4a531ad00a438a602c11bb8dd3bfd03a891e871b4dabaf4d580f2228  LICENSE
```

Comment:
It copied the LICENSE file successfully from the source repo root, not as symlink.